nginx/testcontainers/nginx/__init__.py

```diff
@@ -20,3 +25,17 @@ def __init__(self, image: str = "nginx:latest", port: int = 80, **kwargs) -> Non
         super(NginxContainer, self).__init__(image, **kwargs)
         self.port = port
         self.with_exposed_ports(self.port)
+
+    def start(self) -> 'NginxContainer':
```

Comment:
Were there flaky tests without this change?

Reply:
yes